src/itest/java/uk/gov/companieshouse/companyprofile/steps/companyProfileSteps.java

```diff
@@ -95,8 +140,26 @@ public void theMessageShouldRetryAndError(int retries) {
         assertThat(errors).isEqualTo(1);
     }
 
+    @Then("a DELETE request is sent to the company profile api")
+    public void deleteRequestIsSent() {
+        verify(1, deleteRequestedFor(urlMatching(
+                "/company/00358948")));
+    }
+
     private void countDown() throws Exception {
         CountDownLatch countDownLatch = new CountDownLatch(1);
         countDownLatch.await(5, TimeUnit.SECONDS);
     }
+
+    private void stubPutStatement(int responseCode) {
+        stubFor(put(urlEqualTo(
+                "/company/00358948"))
+                .willReturn(aResponse().withStatus(responseCode)));
+    }
+
+    private void stubDeleteStatement(int responseCode) {
+        stubFor(delete(urlEqualTo(
+                "/company/00358948"))
+                .willReturn(aResponse().withStatus(responseCode)));
+    }
```

Comment:
We need amethod to make sure wiremock shuts down properly after each test is run. this should do it     @After
    public void shutdownWiremock(){
        if (wireMockServer != null)
            wireMockServer.stop();
    }